Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -243,6 +249,8 @@ def formula_or_cask_name(formula_or_cask, full_name: false)
         formula_name(formula_or_cask, full_name: full_name)
       when Cask::Cask
         cask_name(formula_or_cask, full_name: full_name)
+      else
+        T.absurd(formula_or_cask)
```

Comment:
What's this do?

Reply:
If we ever change the type of `formula_or_cask` from

`T.any(Formula, Cask::Cask)`

to say,

`T.any(Formula, Cask::Cask, Thing)`

then Sorbet will give an error since we don't handle the case for if `formula_or_cask` is a `Thing`

(Admittedly, this is a bit redundant since the name `formula_or_cask` itself implies that it can only be a `Formula` or `Cask`)

Reply:
Should I delete?

Reply:
I don't feel strongly either way, up to you! Thanks for explaining ❤️